### PR TITLE
Task/ds 14

### PR DESCRIPTION
**Pull request by sudheerad9**
Added api's for create/update/delete user for admins, serializer for admin access.

### CODE REVIEW

File: accounts/models.py [hunk 1]
```diff
@@ -87,8 +87,8 @@ def has_module_perms(self, app_label):
 		# Simplest possible answer: Yes, always
 		return True
 
-	@property
```
Comment (sudheerad9) [inline accounts/models.py:90]: Was getting an error when trying to modify is_staff so removed it.
https://stackoverflow.com/questions/62504785/typeerror-user-got-an-unexpected-keyword-argument-is-staff-when-using-custo

File: accounts/serializers.py [hunk 2]
```diff
@@ -40,4 +40,39 @@ class UserInfoTransSerializer(serializers.Serializer):
...
+        return obj.first_name + " " + obj.last_name
+
+
+class UserSerializerAdminAccess(serializers.ModelSerializer):
+	password = serializers.CharField(write_only=True)
+	def create(self, validated_data):
+		is_active = validated_data.get('is_active')
+		is_staff = validated_data.get('is_staff')
+		is_superuser = validated_data.get('is_superuser')
+		if not is_active:
```
Comment (tjquinn1) [inline accounts/serializers.py:52]: Just set the default value when trying to get is_active. .get('is_active', False). Also is_active should be set to True by default so there should be no need to set it to true again.

**Review by tjquinn1 — CHANGES_REQUESTED** [review]

File: accounts/models.py (hunk 1 above)
Comment (sudheerad9) [reply accounts/models.py:90]: If the @property method is removed , I was able to modify it.

Comment (sudheerad9) [reply accounts/models.py:90]: I was able to modify is_staff value after removing the @property method.

**Pull request closed without merging**